.github/workflows/delete-v2-review-app.yml

```diff
@@ -106,3 +106,7 @@ jobs:
 
         env:
           DEPLOY_ENV: ${{env.DEPLOY_ENV}}
+      - uses: geekyeggo/delete-artifact@v2
+        with:
+          name: failure-uploads
```

Comment:
Don't we need to add the path as in https://github.com/DFE-Digital/apply-for-teacher-training/pull/8848/files#diff-ecad2183af1362d9d99f4a33e7491c1970c92255c3c318e688ece549ec91fe33R266 ?

Reply:
Isn't that the path it takes the files from? I'll test on this PR once it gets merged.

Reply:
but the path is "path: /app/tmp/failures/" ?